scripts/validate-for-appstore.sh

```
@@ -0,0 +1,338 @@
+
+# Entitlements
+if [ -f "UnaMentis/UnaMentis.entitlements" ]; then
+  print_check "UnaMentis.entitlements exists"
+else
+  print_warning "UnaMentis.entitlements not found"
+  ((++WARNINGS))
+fi
+
+# App Icons
+if [ -d "UnaMentis/Assets.xcassets/AppIcon.appiconset" ]; then
+  ICON_COUNT=$(ls UnaMentis/Assets.xcassets/AppIcon.appiconset/*.png 2>/dev/null | wc -l | tr -d ' ')
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)